src/VsixV3/EditorsPackage/Microsoft.VisualStudio.Editors.swr

```diff
@@ -1,7 +1,7 @@
 use vs
 
 package name=Microsoft.VisualStudio.Editors
-        version=15.0.0.0
+        version=$(Version)
```

Comment:
This is fine for today. @basoundr is out sick today. @basoundr , once you're back, please find the right variable to put here.